packages/release-notes-generator/src/get-info.ts

```diff
@@ -61,7 +55,7 @@ export async function getInfo(changesets: ChangesetsWithoutId) {
 			}
 
 			if (!packageVersions.has(name)) {
-				const version = packageInfo.find((p) => p.name === name)?.version;
+				const version = workspacePackages.find((p) => p.manifest.name === name)?.manifest.version;
```

Comment:
since this line seems to be the same as line 31, should we combine them into a single line upwards?

Reply:
Thanks for spotting this - was a leftover & has been removed now 👍